### PR TITLE
AEA-3870 Releasing with Responsible Practitioner causes 500 error

### PR DESCRIPTION
**Pull request by JackSpagnoliNHS**
## Summary

- Routine Change

### Details

Updates ResponsiblePractitioner conversion to populate AgentPerson information from attached organization if ResponsiblePractitioner is only an organization.

### CODE REVIEW

File: packages/coordinator/src/services/validation/bundle-validator.ts [hunk 1]
```diff
@@ -131,11 +145,12 @@ export function verifyCommonBundle(
...
   bundle: fhir.Bundle,
   practitionerRole: fhir.PractitionerRole,
+  isResponsibleParty: boolean,
   incorrectValueErrors: Array<fhir.OperationOutcomeIssue>,
   accessTokenSDSUserID: string,
   accessTokenSDSRoleID: string
-): void{
-  if(practitionerRole.practitioner) {
+): void {
+  if (practitionerRole.practitioner && !isResponsibleParty) {
```
Comment (MatthewPopat-NHS) [inline packages/coordinator/src/services/validation/bundle-validator.ts:153]: We should validate that it's either a reference or an identifier if it's a responsible party

Comment (JackSpagnoliNHS) [reply packages/coordinator/src/services/validation/bundle-validator.ts]: Updated in eb325ec81a6249a7534ccc70c375f200f43f0128

File: packages/specification/schemas/components/PractitionerRole/CreatePractitionerRole.yaml [hunk 2]
```diff
@@ -18,13 +23,28 @@ properties:
...
+        properties:
+          identifier:
+            oneOf:
+              - $ref: ../../components/_fragments/identifiers.yaml#/components/schemas/DinNumber
+              - $ref: ../../components/_fragments/identifiers.yaml#/components/schemas/GmcNumber
+              - $ref: ../../components/_fragments/identifiers.yaml#/components/schemas/GmpNumber
+              - $ref: ../../components/_fragments/identifiers.yaml#/components/schemas/NmcNumber
+              - $ref: ../../components/_fragments/identifiers.yaml#/components/schemas/GphcNumber
+              - $ref: ../../components/_fragments/identifiers.yaml#/components/schemas/HcpcNumber
+              - $ref: ../../components/_fragments/identifiers.yaml#/components/schemas/ProfessionalCode
```
[the remark this answers is not in the document]
Comment (JackSpagnoliNHS) [reply packages/specification/schemas/components/PractitionerRole/CreatePractitionerRole.yaml:44]: 9153e69fb03f318adc9115dfe6b201d22fc87f83

File: packages/specification/schemas/components/PractitionerRole/CreatePractitionerRole.yaml [hunk 3]
```diff
@@ -18,13 +23,28 @@ properties:
         NHS BSA Spurious Code:
           $ref: ../../components/_fragments/identifiers.yaml#/components/schemas/NhsBsaSpuriousCode
```
[the remark this answers is not in the document]
Comment (JackSpagnoliNHS) [reply packages/specification/schemas/components/PractitionerRole/CreatePractitionerRole.yaml]: b4803da50f8a600d53daeed4635e1cbc448d261b

**Review by MatthewPopat-NHS — APPROVED** [review]

**Pull request merged**